test/distributed/org/apache/cassandra/distributed/test/cql3/SingleNodeTableWalkTest.java

```diff
@@ -460,7 +466,8 @@ public State(RandomSource rs, Cluster cluster)
             {
                 model.factory.regularAndStaticColumns.forEach(mutationGenBuilder::allowEmpty);
             }
-            this.mutationGen = toGen(mutationGenBuilder.build());
+            model.factory.regularAndStaticColumns.forEach(mutationGenBuilder::allowNull);
```

Comment:
was doing empty bytes and normal values, now injecting `null` as well.... this was to try to inline as much of `org.apache.cassandra.cql3.RandomSchemaTest` we could